matlabCode/createBIDS_ieeg_channels_tsv.m

```diff
@@ -26,51 +27,57 @@
 %% make a participants table and save 
 
 
-
 %% required columns
 name = {''}; % Label of the channel, only contains letters and numbers. 
 % The label must correspond to _electrodes.tsv name and all ieeg type channels 
 % are required to have a position. In case of bipolar recordings where the 
 % channels are plugged into the hardware in bipolar manner, the channel name 
-% must be ?Name01-Name02? with a dash between the two electrode names.
+% must be ?Name01-Name02? with a dash between the two electrode names. The
+% reference channel name MUST be provided in the reference column
 
-type = {''}; % Type of channel, see below for adequate keywords in this field 
-% e.g. SEEG, ECOG. 
+type = {''}; % Type of channel e.g. SEEG, ECOG. 
```

Comment:
Comment should be: Type of channel, see below for adequate keywords in this field.